static/js/components/Header.js

```diff
@@ -25,6 +26,7 @@ class Header extends React.Component {
               <img src="/static/images/logo-micromasters.png" width="215" height="40" alt="MIT MicroMasters" />
             </a>
           </Navbar.Brand>
+          <Link to="/users/foobar">FOOBAR</Link>
```

Comment:
this is just for testing, to make it easier to visit a page where I'll get a 404